shimmingtoolbox/load_nifti.py

```diff
@@ -179,11 +179,19 @@ def read_nii(fname_nifti, auto_scale=True):
         if ('Manufacturer' in json_data) and (json_data['Manufacturer'] == 'Siemens') \
                 and (('ImageComments' in json_data) and ("*phase*" in json_data['ImageComments'])
                      or ('ImageType' in json_data) and ('P' in json_data['ImageType'])):
-            # Bootstrap, rescales from -pi to pi
-            if np.amin(image) < 0:
-                image = image * (2 * math.pi / (PHASE_SCALING_SIEMENS * 2))
+            # Rescales from -pi to pi
+            extent = (np.amax(image) - np.amin(image))
+
+            if np.amin(image) < 0 and (0.9 * 2 * PHASE_SCALING_SIEMENS < extent < 2 * PHASE_SCALING_SIEMENS * 1.1):
+                # Siemens' scaling: [-4096, 4095] --> [-pi, pi]
```

Comment:
to be mathematically correct i think it would be
```suggestion
                # Siemens' scaling: [-4096, 4095] --> [-pi, pi)
```